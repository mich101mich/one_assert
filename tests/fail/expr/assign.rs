fn main() {
    let mut x = true;
    one_assert::assert!(x = false);
}
