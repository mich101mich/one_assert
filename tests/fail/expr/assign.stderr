error: Expected a boolean expression, found an assignment. Did you intend to compare with `==`?
 --> tests/fail/expr/assign.rs:3:27
  |
3 |     one_assert::assert!(x = false);
  |                           ^
